cli/pcluster/utils.py

```diff
@@ -164,3 +163,22 @@ def get_instance_vcpus(region, instance_type):
         vcpus = -1
 
     return vcpus
+
+
+def get_supported_os(scheduler):
+    """
+    Return a tuple of the os supported by parallelcluster for the specific scheduler.
+
+    :param scheduler: the scheduler for which we want to know the supported os
+    :return: a tuple of strings of the supported os
+    """
+    return "alinux" if scheduler == "awsbatch" else "alinux", "centos6", "centos7", "ubuntu1404", "ubuntu1604"
```

Comment:
I checked and there isn't another definition (unless I am wrong and don't know that)

I decided to use a tuple because it's immutable and the function calling supported_os and schedulers should not change them. 

Dunno about good python practice though, that's a java style way.